### PR TITLE
Running kubelet v1.26 fixes (deprecated args)

### PR DESCRIPTION
**Pull request by a4099181**
**Reason for PR**:
Deprecated arguments removed: --log-dir=/var/log/kubelet --logtostderr=false

**Requirements**

- [x] Sqaush commits

### CODE REVIEW

**Review by fabi200123 — APPROVED** [review]
I have also tested this. Indeed these arguments does not allow kubelet to start on k8s v1.26. @jsturtevant can you take a look as well?
For me it is /lgtm

**Comment by jsturtevant** [comment]
I see they are marked as deprecated but I don't seem them being removed in the 1.26 docs (https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)? Could you link to where they are removed and causing issues starting?  Otherwise looks good.

**Comment by fabi200123** [comment]
@jsturtevant They said [here ](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components) that `--log-dir`, `--logtostderr` and other flags will become deprecated in future. It also say that [**Kubernetes 1.26 (tentative): GA, deprecated flags get removed**](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components#implementation-history).
Also, @marosset @jayunit100 I believe you should check this one out as well

File: kubeadm/scripts/PrepareNode.ps1 [hunk 1]
```diff
@@ -94,12 +94,17 @@ New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C
...
+$cmd_commands=@("C:\k\kubelet.exe ", '$global:KubeletArgs ', '--cert-dir=$env:SYSTEMDRIVE\var\lib\kubelet\pki ', "--config=/var/lib/kubelet/config.yaml ", "--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf ", "--kubeconfig=/etc/kubernetes/kubelet.conf ", '--hostname-override=$(hostname) ', '--pod-infra-container-image=`"mcr.microsoft.com/oss/kubernetes/pause:3.6`" ', "--enable-debugging-handlers ", "--cgroups-per-qos=false ", '--enforce-node-allocatable=`"`" ', '--resolv-conf=`"`" ')
 [version]$CurrentVersion = $($KubernetesVersion.Split("v") | Select -Index 1)
 [version]$V1_24_Version = '1.24'
 if ($CurrentVersion -lt $V1_24_Version) {
     $cmd_commands = $cmd_commands + "--network-plugin=cni " + "--image-pull-progress-deadline=20m "
 }
+[version]$V1_26_Version = '1.26'
+if ($CurrentVersion -lt $V1_26_Version) {
+    $cmd_commands += ("--log-dir=/var/log/kubelet ", "--logtostderr=false ")
```
Comment (claudiubelu) [inline kubeadm/scripts/PrepareNode.ps1:105]: nit: ideally, it would have been consistent if the ``$V1_24_Version`` check and appending above.

**Comment by claudiubelu** [comment]
> I see they are marked as deprecated but I don't seem them being removed in the 1.26 docs (https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)? Could you link to where they are removed and causing issues starting? Otherwise looks good.

I see in the v1.26 release notes (https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md): ``Legacy klog flags are no longer available. Only -v and -vmodule are still supported. (https://github.com/kubernetes/kubernetes/pull/112120, [@pohly](https://github.com/pohly)) [SIG Architecture, CLI, Instrumentation, Node and Testing]``. The PR linked is merged in v1.26: https://github.com/kubernetes/kubernetes/pull/112120

/lgtm

**Pull request merged**